lms/djangoapps/instructor_task/api_helper.py

```diff
@@ -75,6 +95,36 @@ def _reserve_task(course_id, task_type, task_key, task_input, requester):
     return InstructorTask.create(course_id, task_type, task_key, task_input, requester)
 
 
+def generate_already_running_error_message(task_type):
+    """
+
+    Returns already running error message for given task type.
+    """
+
+    message = ''
+    already_running_message_template = "The {} report is being created. " \
+                                       "To view the status of the report, see Pending Tasks below. " \
+                                       "You will be able to download the report when it is complete."
+    report_types = {
```

Comment:
@noraiz-anwar  Can you use this mapping to extract ` success_status` in the view as well?
What I basically mean is that, pass `report_type` from `api.py` rather than constructing report type in `instractor_task/api.py`. 
e.g. `problem_responses_csv`.

Then, the `SUCCESS_MESSAGE_TEMPLATE` can call this mapping and get the value of the report type message just like `already_running_message_template` is getting. 
Let me know if you have questions.